### PR TITLE
prysmctl: check capella fork for blstoexecutionchange

### PR DESCRIPTION
**Pull request by james-prysm**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

Adding in fork checking to make sure capella is enabled before submitting blstoexecutionchanges as the api endpoint does not throw an error for messages prior to the fork.

### CODE REVIEW

**Review by terencechain — COMMENTED** [review]
Aren't they signed with the genesis fork version these days?

File: cmd/prysmctl/validator/withdraw.go [hunk 1]
```diff
@@ -87,6 +89,13 @@ func callWithdrawalEndpoints(ctx context.Context, host string, request []*apimid
 	if err != nil {
 		return err
 	}
+	fork, err := client.GetFork(ctx, "head")
```
Comment (potuz) [inline cmd/prysmctl/validator/withdraw.go:92]: Isn't it simpler to check that `CapellaForkEpoch` is set and then that current epoch is higher than that?

**Review by potuz — APPROVED** [review]
Looks good to me, just wondering why not simply checking against current epoch instead of the current head fork

**Comment by potuz** [comment]
> Aren't they signed with the genesis fork version these days?

They are but the idea is not making it easier for users to shoot their feet by including them now. And only allow these changes to be post-Capella. Users that insist on flooding the fork should find their way

File: cmd/prysmctl/validator/withdraw.go (hunk 1 above)
Comment (james-prysm) [reply cmd/prysmctl/validator/withdraw.go:92]: yeah that's true, at first I thought I somehow I could compare the output of fork response with version.Capella somehow and winded up on this road 😓

**Pull request merged**